src/spec/EngineSpec.ts

```diff
@@ -3,20 +3,20 @@
 
```

Comment:
Please remove whitespace changes

Reply:
@LoserAntbear We only want changes relevant to feature at hand. Having whitespace changes in unrelated files makes it difficult for us to use git history to see what has changed when for a feature. It makes reviewing more difficult as well.

Git allows you to checkout files back to master `git checkout origin/master -- path/to/file` this could be used on files that don't have changes pertinent to event propagation to revert those whitespace changes, ie: `EngineSpec.ts`, `SoundSpec.ts`, etc.

Normally I'd totally agree with this whitespace cleanup, it just belongs in a separate PR, probably with some tooling (#884) to prevent extraneous whitespace from being committed again.

Reply:
I made new PR, which closes #884. Here it is: https://github.com/excaliburjs/Excalibur/pull/988
I also moved there `Jasmine 2.8.0` type definitions changes and all this whitespace removals and fixes in `EngineSpec.ts` and `SoundSpec.ts`.
But i have to wait for this one to be merged, 'cause i  need Jasmine 2.8.0 features for using in testes for this PR.